src/Home.js

```diff
@@ -113,39 +141,39 @@ class Home extends React.Component {
       }
     }
     for (var i = 0; i < selectedModuleIds.length; i++) {
-        for (var j=0; j < courseInfo.length; j++){
-          if (selectedModuleIds[i]===courseInfo[j].related_module_id)
-          selectedCourseData.push(courseInfo[j]);
-        }
+      for (var j=0; j < courseInfo.length; j++){
+        if (selectedModuleIds[i]===courseInfo[j].related_module_id)
```

Comment:
I'd advise to not omit braces for if statements, for loops, etc.

The reason is the following:
It is easy to overread that the line after the if clause belongs to it, so it is easy to assume that the newly added line still belongs to the if case even though it doesn't.